### ISSUE TITLE
(resolved) HTGQL NodeJs tutorial - env variables

### ISSUE THREAD

**Issue by ScorpIan555** [howtographql/howtographql#1025]
Ref: https://github.com/prisma/prisma-examples/issues/303   (bensonodede's reponse toward the bottom)

As I am hoping to jump from this project into a "real" project of my own, I've been trying out changes I'd need for a production-ready project.  One issue I had was figuring out the yml string interlopation of the prisma.yml & docker-compose.yml

With docker-compose.yml, I was able to easily use the ${VARIABLE_NAME} sytax, no problem -- with the .env that the tutorial instructions tell us to create in the ./prisma/ directory.

With prisma.yml, referencing the issue above, I had to move my env variable definitions up to the root folder.  I think this is because the 'prisma deploy' CLI command generates the /generated/ directory within the project's /src/ directory.  In the file system tree, /src/ & /prisma/ are at the same level.  

If I'm understanding the explanation correctly, the /src/generated/index.js file is looking for a .env file first in the /src/generated/ directory, then up one level.  I was able to resolve this by creating a new .env file.  

It's might be a good idea to clarify the part of the tutorial explaining the declaration of environmental variables a little to incorporate this.

Cheers, thanks for a great tutorial.

**Comment by chapsu01** [comment]
thanks a lot. 
TLDR;
inside `server/src/generated`,  run `touch .env`
